crypto/onetimesig.go

```diff
@@ -322,7 +322,7 @@ func (v OneTimeSignatureVerifier) Verify(id OneTimeSignatureIdentifier, message
 	return batchVerificationImpl(
 		[][]byte{HashRep(batchID), HashRep(offsetID), HashRep(message)},
 		[]PublicKey{PublicKey(v), PublicKey(batchID.SubKeyPK), PublicKey(offsetID.SubKeyPK)},
-		[]Signature{Signature(sig.PK2Sig), Signature(sig.PK1Sig), Signature(sig.Sig)},
+		[]Signature{Signature(sig.PK2Sig), Signature(sig.PK1Sig), Signature(sig.Sig)}, nil,
```

Comment:
why nil?

Reply:
This call to `batchVerificationImpl` does not need the indexes of the failed signatures. Passing a nil will not fill this information.
I though this should be fine, since this is an internal function.

Reply:
ah, because of line wrapping I read `Signature(sig.Sig)` -> `nil`